examples/nextjs-swc/src/app/[lang]/app-router-demo/page.tsx

```diff
@@ -1,7 +1,7 @@
 import { HomePage } from '../../../components/HomePage'
-import { initLingui } from '../../../initLingui'
+import { initLingui, PageLangParam } from "../../../initLingui"
```

Comment:
Please make the quote format consistent here and in a few other places below. Some imports use `"`, some `'`

Reply:
interesting, it should be covered by prettier, why it doesn't